Design.md

```diff
@@ -0,0 +1,26 @@
+Overview
+    - This app gives players and teams an environment where they can seamlessly review and breakdown film from games / practices all in one place. It can be used for both internal development and for detailed scouting on future opponents.
+
+Features:
+1. Team Specific Accounts
+    - User notes with this team affiliation are all put in one place.
+    - This means scouting can be done without worrying about another team user seeing it.
+2. User Roles
+    - Users who are verified with a team and can post to that team's account
+    - Users who are not verified with a team and will only be posting on their personal account
+3. Watch Games in App
+    - In a perfect world, the video is playing in the same browser as the app is running.
+    - However, due to cose we might have to use external browser like youtube to run videos.
+        - If this is the case then make it easy to link film clip with timestamp of video
+4. Create Film Clips
+    - While watching a video, user is able to clip part of a video and make a note about what happens
+    - If this user is verified with a team, when another team member is watching the same video they will be able to see the note.
+    - Film clips are kept in a team-wide / user-wide database that can be searched / sorted by keywords (ie. players involved, topic discussed (ie. deep defense), teams, etc...)
+5. Allow Highlights to be Shared
+    - Plays that are deemed highlights while watching can easily be shared
+    - User has the option to include notes along with clip or if they only want to share the clip
+
+Rough Design
+![Home/Game Review](design-draft-images/home:game-review.jpg)
+![Search Home / Team Profile (other team)](design-draft-images/search-home:team-profile.jpg)
```

Comment:
Search Home:
1. Should we keep the search bar here or should we have a search bar like, everywhere, that when you click "enter" or whatever it takes you to this page?
2. What is the "Game Review Board" that this button takes you to? If that another page, can you upload a pic of the design for it? If it's the game review page from before, I feel like we should add some more stuff to it to make it a full-fledged game review thing (like maybe not just having all the notes be highlighted in the video, but also having them displayed below the video in a table the user can easily click on and filter etc.) rather than just a video with a form.
3. Should we link out to the game review thing, or embed the specific clip right in the search result so they can easily watch? It might not be that useful and might be more annoying than anything...

Team Profile (other team):

1. This page is missing some detail -- For those games, does clicking on them take you to that game in the game review page? Does clicking on a player bring up a list of all the plays that player has been involved in?
2. Does "Scouting Tab" need to take up that much space in the middle of the page? I feel like if you're trying to tell the user what page they're on, it makes more sense to put it up in the nav bar up top.
3. Technical question, not relevant right now but interesting to think about: Can we get this data from AUDL somehow? That would be a lot less painful than asking each team to auto-populate their stuff. I can see there's some stuff in the python package we can probably crib.

Reply:
Search Home

1. I totally agree, I do want there to be some easy way to search again or add filters to their search when they get to this page though. I am going to dramatically decrease the size of the search bar for sure.
2. Add another feature for how search results will work, see (#7) of code
3. I think we link out the game review thing personally it will be too clunky in my opinion.

Team Profile
1. Made a lot of changes to this page in second draft of the layout. Clicking on a player in their rosters automatically creates a search for that player and sends user to the search results page with a list of that players plays populated. The game link will take you directly to that game's film review board.
2. The scouting tab was made into a message board system. Where you can write notes unrelated to film clips that describe the team or certain players in particular.
3. We probably could for AUDL but I also want to include club teams in here as well and the stats and stuff are a little less documented sadly.